### ISSUE TITLE
Parse `profile` messages that contain a `list` of `^` delimited profiles.

### ISSUE THREAD

**Issue by stephenhouser** [stephenhouser/node-red-contrib-flexradio#6]
This issue may already be resolved.

I would like to have 'profile' messages injected properly into the flows so that I can use the data from those profiles in my work.

Example profile response:
```
S6190EABE|profile mic list=Default^Default FHM-1^Default FHM-1 DX^Default FHM-2^Default FHM-2 DX^Default FHM-2 ESSB^Default FHM-3^Default FHM-3 DX^Default FHM-3 ESSB^Default HM-Pro^Default PR781^Default PR781 ESSB 3_2k^Default ProSet HC6^Inrad M629^Inrad M650^RadioSport DX M207^RadioSport DX M208^RadioSport DX M350-ADJ^RadioSport DX M360/EM56^RadioSport WIDE M207^RadioSport WIDE M208^RadioSport WIDE M350-ADJ^RadioSport WIDE M360/EM56^RTTYDefault^
```

**Comment by stephenhouser** [comment]
From capture of SmartSDR connection. These generate profile strings that are not parsed properly.

```
C10|profile global info
C11|profile tx info
C12|profile mic info
C13|profile display info
```

**Comment by stephenhouser** [comment]
There are a number of other responses from the radio that are not parsed correctly. Working on rewriting the parser.

**Comment by stephenhouser** [comment]
fixed in commit b5c285a4df46ecd3e779e683af14884371b9b266
